### PR TITLE
added groupedEach helper and test

### PR DESCRIPTION
**Pull request by robsilva**
added groupedEach helper and test

### CODE REVIEW

File: lib/array.js [hunk 1]
```diff
@@ -206,6 +206,46 @@ helpers.forEach = function(array, options) {
...
+ * @param {Array} `array`
+ * @param {any} `value`
+ * @param {Object} `options`
+ * @return {String}
+ * @block
+ * @api public
+ */
+
+helpers.groupEach = function (every, context, options) {
+  var out = "", subcontext = [], i;
```
Comment (jonschlinkert) [inline lib/array.js]: would you mind using single quotes please? everything else looks great, thanks.

**Comment by robsilva** [comment]
All set. Thank you

File: lib/array.js [hunk 2]
```diff
@@ -206,6 +206,46 @@ helpers.forEach = function(array, options) {
...
+ * @name .inArray
+ * @param {Array} `array`
+ * @param {any} `value`
+ * @param {Object} `options`
+ * @return {String}
+ * @block
+ * @api public
+ */
+
+helpers.groupEach = function (every, context, options) {
```
Comment (doowb) [inline lib/array.js]: The arguments should be `(context, every, options)` to match the other array helpers.
Also, change `context` to `array` to match the other helpers.
The code comments above need to be updated with the correct name and parameter information.

**Comment by robsilva** [comment]
@doowb - Sorry about the delay. These changes have been made. Thanks!

**Comment by jonschlinkert** [comment]
@doowb have you had a chance to review?

**Comment by robsilva** [comment]
Can this be merged soon, please? :)

**Comment by doowb** [comment]
Sorry for the delay... the changes look good. I'll get this merged and published this weekend.

**Pull request merged**